### ISSUE TITLE
🐛 BUG: hyperdrive localConnectionString changed to sslmode=prefer after updated to 3.86.1

### ISSUE THREAD

**Issue by j4tmr** [cloudflare/workers-sdk#7255]
### Which Cloudflare product(s) does this pertain to?

Wrangler

### What version(s) of the tool(s) are you using?

3.86.1

### What version of Node are you using?

20.10.0

### What operating system and version are you using?

Mac Sonoma 14.6.1

### Observed behavior
hyperdrive localConnectionString not working
sslmode=disable in 3.85.0

**Comment by edmundhung** [comment]
Hi @j4tmr ! Yes, we’ve [updated localConnectionString to use `sslmode=prefer`](https://github.com/cloudflare/workerd/pull/3037) recently. This change should allow connections to use SSL if available while still supporting unencrypted connections for local test databases.

If you run into any issues because of this change, could you share more details, such as any error messages you’re seeing? That would help us investigate further. Thanks!

**Comment by j4tmr** [comment]
`The server does not support SSL connections`. While i'm using supabase locally, with configuration

```
[[hyperdrive]]
binding = "xxx"
id = "xxx"
localConnectionString = "postgresql://postgres:postgres@127.0.0.1:54322/postgres"
```

**Comment by sambarnes** [comment]
seeing the same thing as j4tmr -- appears we're both using supabase

this worked previously but no longer does

local workaround is doing this for dev environment:
`c.env.HYPERDRIVE_SUPABASE.connectionString.replace('sslmode=prefer', 'sslmode=disable')`

not great but unblocks for now

**Comment by ivanfgm** [comment]
Same here with a remote timescale instance (CONNECT_TIMEOUT), could only get it working replacing prefer with disable same as @sambarnes

**Comment by emily-shen** [comment]
Hiya, there will be a fix coming in a couple of hours https://www.cloudflarestatus.com/incidents/r6ltkr1t0vkr

**Comment by emily-shen** [comment]
Hey this should be fixed in the latest release 3.92.0 now
